src/app/pages/TermsPage/components/terms/terms.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="terms">
+    <div class="page-title">
+        <h1>
+            Terms and conditions
+        </h1>
+    </div>
+    <div class="row">
+        Demo content

```

Comment:
As per the bootstrap spec, ".row"s should always have a ".col" elements as children. So you could add a "col-lg-12" div around "Demo content" so the people who add the content don't forget to. A simple reason why a col-lg-12 element is needed is the fact that rows have negative horizontal margins and the columns account for those with their own paddings

Reply:
Thanks, updated!